src/test/scala/com/fulcrumgenomics/commons/io/IoTest.scala

```diff
@@ -193,4 +194,27 @@ class IoTest extends UnitSpec {
     val actual = Io.readBytesFromResource("/com/fulcrumgenomics/commons/io/to-bytes-from-resource-test.bin")
     actual shouldBe expected
   }
+
+  "Io.toInputStream" should "open a file for gzip writing if it ends with .gz" in {
+    val text = "This is a stupid little text fragment for compression. Yay compression!"
+    val in   = Seq(text, text, text)
+    val f = Io.makeTempFile("test", ".gz")
+    Io.writeLines(f, in)
+
+    // Manually read it back as gzipped data
+    val reader = new BufferedReader(new InputStreamReader(new GZIPInputStream(new FileInputStream(f.toFile))))
+    val out = Seq(reader.readLine(), reader.readLine(), reader.readLine())
+    out shouldBe in
+    reader.close()
+  }
+
+  "Io.toOutputStream" should "round trip data to a gzipped file if it ends with .gz" in {
+    val text = "This is a stupid little text fragment for compression. Yay compression!"
+    val in   = Seq(text, text, text)
+    val f = Io.makeTempFile("test", ".gz")
```

Comment:
align = here and above for practice (looks like wasn't in fgbio)?

Reply:
Correct, I am a victim of copy paste formatting: https://github.com/fulcrumgenomics/fgbio/blob/b84524e132e5033a0bbe5c6dd661b1b2681c3bdb/src/test/scala/com/fulcrumgenomics/util/IoTest.scala#L40

Reply:
fixed in latest changes